templates/header.html.ep

```diff
@@ -4,8 +4,12 @@
       <ul class="pure-menu-list">
         <li class="pure-menu-item"><%= link_to 'Home', 'landing_page', class => 'pure-menu-link' %></li>
         <li class="pure-menu-item"><%= link_to 'Travel', 'conference.page', {page => 'travel'}, class => 'pure-menu-link' %></li>
+      % if (session 'uid') {
+        <li class="pure-menu-item"><%= link_to 'Profile', 'user.profile', class => 'pure-menu-link' %></li>
+      % } else {
+        <li class="pure-menu-item"><%= link_to 'Login', 'user.profile', class => 'pure-menu-link' %></li>
+      % }
```

Comment:
This could probably be simplified as `link_to +(session 'uid' ? 'Profile' : 'Login'), 'user.profile' ...` if you like